docs/adal-connectivity.md

```diff
@@ -0,0 +1,57 @@
+# adal-connectivity
+Android library to verify if the device has an Internet connection and what it's type is!
+
+### Download
+
+Gradle:
+
+```gradle
+dependencies {
+  compile 'com.massivedisaster.adal:adal-connectivity:0.1.15'
+}
+```
+### Usage
```

Comment:
@paribeiro10 the Usage should have activity/fragment/application/service code for the user to understand how to call this utility methods. I suggest a more simple usage such as calling the `checkConnectivity` and show its result in a `Toast`, for example.

Reply:
Made changes to the class to make it more clear for this purpose.
I can remove the `TextView` and leave the Toast only. Please advise.

Reply:
Your call.